package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "private": "true",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest --watch *.js",
```

Comment:
- [ ] When I run 'npm test' this happens. Please fix it or if you installed other necessary local dependencies let me know in the comments.
![Screen Shot 2022-02-23 at 15 11 24](https://user-images.githubusercontent.com/67211919/155409400-5702fa6c-400b-4c54-931c-14aa2dd4295c.png)